a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -428,30 +428,35 @@ def _get_patched_ve_ip(self, ve_ip):
         ve_ip = '.'.join(octets)
         return a10_utils.merge_host_and_network_ip(self._subnet.cidr, ve_ip)
 
-    def tag_interface(self, create_vlan_id, vlan_id, ifnum, ve_info, vlan_subnet_id_dict):
+    def tag_interface(self, is_trunk, create_vlan_id, vlan_id, ifnum, ve_info,
+                      vlan_subnet_id_dict):
         if vlan_id not in vlan_subnet_id_dict:
             LOG.warning("vlan_id %s not in vlan_subnet_id_dict %s", vlan_id,
                         vlan_subnet_id_dict)
             return None
 
         self.get_subnet_and_mask(vlan_subnet_id_dict[vlan_id])
 
-        if not self.axapi_client.interface.ethernet.exists(ifnum):
-            LOG.error("ethernet interface %s does not exist", ifnum)
-            raise
-
-        eth = self.axapi_client.interface.ethernet.get(ifnum)
-        if 'action' not in eth or not eth['action'] == 'disable':
-            LOG.warning("ethernet interface %s not enabled, enabling it", ifnum)
-            self.axapi_client.interface.ethernet.update(ifnum, enable=True)
+        if not is_trunk:
+            if not self.axapi_client.interface.ethernet.exists(ifnum):
+                LOG.error("ethernet interface %s does not exist", ifnum)
+                raise
+            eth = self.axapi_client.interface.ethernet.get(ifnum)
+            if 'action' not in eth or not eth['action'] == 'disable':
```

Comment:
As we found during debugging, this can be
`if 'action' not in eth or eth['action'] == 'disable':`

Reply:
Done.